webpack.dev.config.ts

```diff
@@ -36,6 +38,29 @@ module.exports = {
   plugins: commonWebpackConfig.plugins,
   module: {
     rules: [{
+      test: /\.svg$/,
+      use: ['raw-loader']
+    },
+    {
+      test: /\.css$/,
```

Comment:
Please use the more specific regex here. (https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/advanced-setup.html#webpack-configuration) `/ckeditor5-[^/]+\/theme\/[\w-/]+\.css$/`

Reply:
Done Thanks!